admin_manual/configuration_server/background_jobs_configuration.rst

```diff
@@ -64,6 +64,8 @@ Using the operating system cron feature is the preferred method for executing
 regular tasks.  This method enables the execution of scheduled jobs without the
 inherent limitations the Web server might have.
 
+.. note:: Please make sure that your cron.php file has executable permissions and is owned by www-data
```

Comment:
If you use the command in the documentation, you execute `php`, so `cron.php` does not have to be executable itself, no?

Reply:
Yes, you're right. Made the required changes 😃